TombEngine/Objects/TR3/tr3_objects.cpp

```diff
@@ -332,6 +333,20 @@ static void StartEntity(ObjectInfo* obj)
 		obj->SetHitEffect();
 	}
 
+	obj = &Objects[ID_GUN_TURRET];
+	if (obj->loaded)
+	{
+		obj->Initialize = InitialiseGunTurret;
+		obj->control = GunTurretControl;
+		obj->collision = CreatureCollision;
```

Comment:
I know the turret use to be on ceiling and are unlikely for Lara to actually reaches to collide with her, but I though would be more natural to have a collision rutine instead of having her just traspassing it. That's why I added the standard one.